### ISSUE TITLE
Enhancement Request: Changed Event

### ISSUE THREAD

**Issue by adamlonsdale** [aritchie/acr-xamarin-forms#133]
Hi,

Is it possible to implement a Changed event similar to how a Text Box has TextChanged etc. so I can pick up when the user has actually done the signature rather than relying on them pressing a button? 

Thanks

**Comment by aritchie** [comment]
I don't think this would be feasible in the current design as it would need to be a continuous pull mechanism.  The calls to either getimage or getdrawpoints isn't something you want to poll often.
